composer.json

```diff
@@ -14,7 +14,7 @@
   ],
   "require": {
     "php": "^7.1",
-    "phpunit/phpunit": "^6.5",
+    "phpunit/phpunit": "^7.5",
```

Comment:
I think this should be as per #2 i.e. `^6.5 || ~7.0`

Reply:
Done